### PR TITLE
Added download time on each file.

### PR DESCRIPTION
**Pull request by IgnacioPro**
Added elapsed time for download per file.

### CODE REVIEW

**Review by jsanemet — APPROVED** [review]
Looks good.

File: pkg/bucketaws/setup.go [hunk 1]
```diff
@@ -113,12 +113,15 @@ func DownloadFromBucket(svc *s3.S3, dwn *s3manager.Downloader, dates []string, b
...
 					logrus.Info("Downloading ", fileName)
+					start := time.Now()
 					_, err := dwn.Download(
 						fooFile,
 						&s3.GetObjectInput{
 							Bucket: aws.String(viper.GetString("bucket")),
 							Key:    aws.String(*item.Key),
 						})
+					duration := time.Since(start)
+					logrus.Info("Download took: ", duration.Round(10000000))
```
Comment (cescgina) [inline pkg/bucketaws/setup.go]: I would advise against having naked numbers like this. Personally, at first glance I find it difficult to tell to which precision will it be rounded to. Maybe use `time.Second`, `time.Minute`, or similar?

Comment (IgnacioPro) [reply pkg/bucketaws/setup.go]: Good point, changed.

**Review by cescgina — APPROVED** [review]
LGTM

**Pull request merged**